src/Sulu/Bundle/ContactBundle/Entity/AccountRepository.php

```diff
@@ -88,7 +88,7 @@ public function findAccountById(int $id, bool $contacts = false): ?AccountInterf
                 ->addSelect('mainContact')
                 ->addSelect('categories')
                 ->addSelect('categoryTranslations')
-                ->addSelect('partial tags.{id, name}')
```

Comment:
seems to fail on orm 3 see https://github.com/doctrine/orm/issues/8471#issuecomment-2002116608